src/Manager.js

```diff
@@ -0,0 +1,286 @@
+const { EventEmitter } = require("events");
+const merge = require("deepmerge");
+const { writeFile, readFile, exists } = require("fs");
+const { promisify } = require("util");
+const writeFileAsync = promisify(writeFile);
+const existsAsync = promisify(exists);
+const readFileAsync = promisify(readFile);
+const Config = require("./Config.js");
+const User = require("./User.js");
+const { defaultManagerOptions, defaultConfig } = require("./Constants.js");
+class VoiceManager extends EventEmitter {
+  constructor(client, options, init = true) {
+    super();
+    if (!client) throw new Error("Client is a required option.");
+    this.client = client;
+    this.ready = false;
+    this.users = [];
+    this.configs = [];
+    this.options = merge(defaultManagerOptions, options);
+    if (init) this._init();
+  }
+
+  async updateConfig(guildID, updatedConfig) {
```

Comment:
```suggestion
 updateConfig(guildID, updatedConfig) {
```
no use for async? It is redundant

Reply:
Yep tysm!